sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Metrics/Metrics.cs

```diff
@@ -41,11 +41,14 @@ public Metrics(SyntaxTree tree) : base(tree)
             }
         }
 
-        public override int GetExecutableLinesCount()
+        public override int ExecutableLinesCount

```

Comment:
Feel free to ignore it:
`public override int ExecutableLinesCount => new ExecutableLinesWalker().Visit(tree.GetRoot()).ExecutableLineCount;`

Reply:
Good idea, but `Visit` returns void. I could add a new method, but it's not worth it.